### ISSUE TITLE
Deploy on Vercel

### ISSUE THREAD

**Issue by taooceros** [Flow-Launcher/docs#4]
I think Github Pages doesn't work well in China. Is it possible to deploy our document in vercel instead? They provide a much better cdn for accessing in China.

**Comment by jjw24** [comment]
Never used it before, is it possible to deploy it with a CD while having the github pages as well?

**Comment by taooceros** [comment]
I think so. As what I have tried before, it can be automatically deployed like github pages by linking the repo with them.

**Comment by jjw24** [comment]
Does that mean also we will have two site urls?

**Comment by jjw24** [comment]
When we update the pages, we still update it in github repo right?

**Comment by taooceros** [comment]
> Does that mean also we will have two site urls?

Yes, but I think vercel has a cdn so their page can be accessed around the world well.

> When we update the pages, we still update it in github repo right?

Yes, I have deployed one before. Updating the repo will automatically trigger their page update just like github page.

**Comment by taooceros** [comment]
It seems that I have the access to do it, so if you want, I can do that and you can check the result.

**Comment by jjw24** [comment]
yes please :)

**Comment by taooceros** [comment]
Request Sent.

**Comment by jjw24** [comment]
👌 done

**Comment by taooceros** [comment]
> 👌 done

Seems that they require Pro plan for Teams work like us, so it will not be possible to do it right now. But they also may have a discount or free plan for open-source projects, so let's wait for their reply for that.

**Comment by jjw24** [comment]
Legend, thanks for getting in touch with them

**Comment by taooceros** [comment]
@jjw24 https://vercel.com/support/articles/can-vercel-sponsor-my-open-source-project
What do you think about that?
